CombineExpectations.podspec

```
@@ -0,0 +1,24 @@
+Pod::Spec.new do |s|
+  s.name = 'CombineExpectations'
+  s.version = '0.4.0'
+  s.summary = 'Utilities for tests that wait for Combine publishers'
+  s.homepage = 'https://github.com/groue/CombineExpectations'
+  s.license = { :type => 'MIT', :file => 'LICENSE' }
+  s.author = { 'Gwendal Roué' => 'https://github.com/groue' }
+  s.source = { :git => 'https://github.com/groue/CombineExpectations.git', :tag => s.version.to_s }
```

`s.version` is already a string.

```suggestion
  s.source = { :git => 'https://github.com/groue/CombineExpectations.git', :tag => s.version }
```